ddpui/api/airbyte_api.py

```diff
@@ -66,7 +73,7 @@ def get_airbyte_source_definitions(request):
             if source_def["name"] in allowed_sources.split("|")
         ]
     logger.debug(res)
-    return res["sourceDefinitions"]
+    return AirbyteSourceDefinitionResponse(sourceDefinitions=res["sourceDefinitions"])
```

Comment:
You will need to look at the fields in `sourceDefinitions` and then define your schema. It has more fields that just `sourceId` & `name`. Hit the api & see what you get, based on which you can define your schema. 

Also it would be nice to use ninja schema (which is built on top of pydantic) instead of BaseModel, just to keep things consistent.